client-spark/spark-2/src/main/java/org/apache/spark/shuffle/celeborn/SparkShuffleManager.java

```diff
@@ -136,17 +136,25 @@ public <K, V, C> ShuffleHandle registerShuffle(
   }
 
   @Override
-  public boolean unregisterShuffle(int appShuffleId) {
-    if (sortShuffleIds.contains(appShuffleId)) {
-      return sortShuffleManager().unregisterShuffle(appShuffleId);
+  public boolean unregisterShuffle(int shuffleId) {
+    if (sortShuffleIds.contains(shuffleId)) {
+      return sortShuffleManager().unregisterShuffle(shuffleId);
```

Comment:
Why this rename ? IIRC it was changed to `appShuffleId` to distinguish between applications (spark's) shuffle id and celeborn's shuffle id.

Reply:
Prefer keep the name too.

Reply:
Initially, I thought It would be better to change back to shuffle id considering clientFetchThrowsFetchFailure can be set to false
But after giving it some more thought, It is always app shuffle id regardless of whether clientFetchThrowsFetchFailure is set to false
I agree that keeping the original approach is better